mono_repo/lib/src/travis_config.dart

```diff
@@ -89,7 +89,20 @@ ${ignoredKeys.map((k) => '    $k').join('\n')}'''));
       processException(matrix['allow_failures'] as y.YamlList, allowFailures);
     }
 
-    var beforeScript = travisYaml['before_script'] as String;
+    var beforeScriptValue = travisYaml['before_script'];
+
+    List<String> beforeScript;
```

Comment:
Please initialize to an empty list. 

Using a null value for collections leads to all sorts of problems - i.e. what is the difference between null and empty list?

Reply:
@alorenzen – PTAL